app/models/participant_profile/ecf.rb

```diff
@@ -75,10 +75,8 @@ def ecf?
     end
 
     def latest_induction_record_for(cpd_lead_provider:)
-      induction_records
-        .joins(induction_programme: { partnership: { lead_provider: [:cpd_lead_provider] } })
-        .where(induction_programmes: { partnerships: { lead_providers: { cpd_lead_provider: } } })
-        .latest
+      # FIXME: should this method return nil if cpd_lead_provider.nil? or call FindBy regardless (FindBy won't consider a nil lead_provider)
```

Comment:
Good question I think we should return `nil` to preserve the previous behaviour  and not call `FindBy` regardless

Reply:
Could we also reinstate everywhere we swapped out `latest_induction_record_for` for `FindBy` to make sure we're  using this guard/wrapper?

Reply:
Thanks for confirming, I didn't change the interface so all that should be needed is a guard in these methods